daprdocs/content/en/operations/components/pluggable-components/pluggable-components-discovery.md

```diff
@@ -0,0 +1,212 @@
+        dapr.io/app-id: "my-app"
+        dapr.io/enabled: "true"
+        dapr.io/sidecar-listen-addresses: "0.0.0.0"
+    spec:
+      volumes: ## required, the sockets volume
+        - name: dapr-unix-domain-socket
+          emptyDir: {}
+      containers:
+        ### --------------------- YOUR APPLICATION CONTAINER GOES HERE -----------
+        ##
+        ### --------------------- YOUR APPLICATION CONTAINER GOES HERE -----------
+        ### This is the pluggable component container.
+        - name: component
+          volumeMounts: # required, the sockets volume mount
+            - name: dapr-unix-domain-socket
+              mountPath: /dapr-unix-domain-sockets
+          image: YOUR_IMAGE_GOES_HERE:YOUR_IMAGE_VERSION
+          env:
+            - name: DAPR_COMPONENTS_SOCKETS_FOLDER # Tells the component where the sockets should be created.
+              value: /dapr-unix-domain-sockets
+```
+
+Great, do not apply the Deployment yet, let's add one more configuration: The Component Spec.
+
+## Step 4: Declaring a gRPC-based Pluggable Component
+
+gRPC-based Pluggable Componets are defined using the [Component Spec]({{< ref component-schema.md >}}) and its `type` is derived from the socket name (without the file extension).
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: prod-mystore
+spec:
+  type: your_component_type.your_socket_goes_here
+  version: v1
+  metadata:
+scopes:
+  - backend
+```
+
+You should also [scope]({{< ref component-scopes >}}) your component to make sure that only the target configured application will try to connect with the pluggable component since it will only be running in its Deployment, otherwise the runtime will fail when initializing the component.
+
+> Note: Replace `your_socket_goes_here` by your component socket name without any extension, and `your_component_type` by your component type.
+> That's it! **[Apply the created manifests to your kubernetes cluster](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-apply)**, and then you are able to call the statestore APIs via Dapr API,
+
+> Use [kubernetes pod forwarder](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to access the Daprd runtime.
+> See it working by running
+
+```shell
+curl -X POST -H "Content-Type: application/json" -d '[{ "key": "name", "value": "Bruce Wayne", "metadata": {}}]' http://localhost:$PORT/v1.0/state/prod-mystore
+```
+
+Retrieve the value
+
+```shell
+curl http://localhost:$PORT/v1.0/state/prod-mystore/name
+```
+
+> replace $PORT by Dapr http port
+
+{{% /codetab %}}
+{{< /tabs >}}
+
+[uds]: https://en.wikipedia.org/wiki/Unix_domain_socket
+
+## Next Steps
+
+- Getting started with your own .NET Pluggable Component using our [sample code](https://github.com/mcandeia/dapr-samples/tree/sample/pluggable-components/pluggable-components-dotnet-template)
```

Comment:
Waiting for the PR to be merged so that I can change the link here
```suggestion
- Getting started with your own .NET Pluggable Component using our [sample code](https://github.com/mcandeia/dapr-samples/tree/sample/pluggable-components/pluggable-components-dotnet-template)
```